x-pack/plugins/observability_solution/apm/server/routes/transactions/get_transaction_by_name/index.ts

```diff
@@ -47,8 +81,43 @@ export async function getTransactionByName({
           ]),
         },
       },
+      fields: [
```

Comment:
this can actually be a very small list, it's just used for a redirect iirc, so it just needs a couple of fields.